server.go

```diff
@@ -155,13 +166,12 @@ func (s *Server) Listen() {
 	}
 
 	go func() {
-		s.statusLogger.Info("starting server on " + s.addr)
```

Comment:
Because I saw no reason for it. Was just more complicated. It is about logging. So there is one logger needed. What is printed using a logger depends only on the context/method it uses the logger.